cmd/initContainer/main.go

```diff
@@ -211,14 +191,6 @@ func acquireLeader(ctx context.Context, kubeClient kubernetes.Interface) error {
 
 func executeRequest(client dclient.Interface, kyvernoclient kyvernoclient.Interface, req request) error {
 	switch req.kind {
-	case policyReportKind:
-		return removePolicyReport(client, req.kind)
-	case clusterPolicyReportKind:
-		return removeClusterPolicyReport(client, req.kind)
-	case reportChangeRequestKind:
-		return removeReportChangeRequest(client, req.kind)
-	case clusterReportChangeRequestKind:
-		return removeClusterReportChangeRequest(client, req.kind)
```

Comment:
How we are going to handle the upgrade to the 1.8.0 release ?  are the we going to cleanup the RCR/ CRCR resources while upgrading ?

Reply:
They will go away when deleting the CRD.